language/stdlib/src/lib.rs

```diff
@@ -19,18 +20,27 @@ lazy_static! {
 ///
 /// The order the modules are presented in is important: later modules depend on earlier ones.
 pub fn stdlib_modules() -> &'static [VerifiedModule] {
-    &*STDLIB_MODULES
+    &*ANNOTATED_STDLIB.0
+}
+
+/// Returns a reference to the source maps for the standard library.
+///
+/// The order of the modules returned in this follows the same order as the modules returned in the
+/// stdlib_modules.
+pub fn stdlib_source_map() -> &'static [ModuleSourceMap] {
```

Comment:
Is there Rust code that would want to call this?

Reply:
Not currently, but my thinking is that there would be (e.g. Clark's code). And, having these prebuilt for the stdlib makes sense since the stdlib is already compiled as well.